.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
           - s3
           - s3-private
           - s3-public
+          - script/database_role
```

Comment:
NIT: should we name it `database-roles`
* plural, since we can create scripts for different roles (we usually have a read-only role and and application-role)
* dashes for consistency (see `vpc-peering`, `s3-*`, `iam-for-*`)